src/core_modules/capture-core/components/WidgetEventSchedule/hooks/useDetermineSuggestedScheduleDate.ts

```diff
@@ -121,9 +122,8 @@ export const useDetermineSuggestedScheduleDate = ({
         }),
     ];
     const suggestedDate = scheduleDateComputeSteps.reduce((currentScheduleDate, computeScheduleDate) =>
-        (!currentScheduleDate ? computeScheduleDate() : currentScheduleDate)
```

Comment:
Rollback to `const suggestedDate = scheduleDateComputeSteps.reduce((currentScheduleDate, computeScheduleDate) =>
        (!currentScheduleDate ? computeScheduleDate() : currentScheduleDate)`